phoenix-tracing-webapp/src/main/java/org/example/Tracing.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.example;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.AbstractHandler;
+
+
+/**
+ *
+ * tracing web app
+ *
+ * @since 4.5.5
+ */
+
+public class Tracing extends AbstractHandler {
+    public void handle(String target,
+                       Request baseRequest,
+                       HttpServletRequest request,
+                       HttpServletResponse response)
+        throws IOException, ServletException {
+        response.setContentType("text/html;charset=utf-8");
+        response.setStatus(HttpServletResponse.SC_OK);
+        baseRequest.setHandled(true);
+        response.getWriter().println("<h1>Tracing Table</h1>");
+    }
+
+    public static void main(String[] args) throws Exception {
```

Comment:
You'll need to decouple the main method from the handler implementation. To make this work with standard hadoop process lifecycle management, there's some other details to follow (like using the `Tool` and `ToolRunner` APIs). I've implemented a structure you can follow in the queryserver. See https://github.com/apache/phoenix/blob/4.4-HBase-0.98/phoenix-server/src/main/java/org/apache/phoenix/queryserver/server/Main.java for an example of what your entry point will need to look like.